application/controllers/gcp_survey_response.py

```diff
@@ -86,36 +84,48 @@ def add_survey_response(self, case_id: str, file_contents, file_name: str, surve
                 bound_log.exception("A non-timeout error was raised when publishing to pubsub", payload=payload)
                 raise SurveyResponseError()
 
-    def put_file_into_gcp_bucket(self, message: dict):
+    def put_file_into_gcp_bucket(self, message: str, filename: str):
```

Comment:
is the str type hint correct here? I'd expect it to be a bytes array?

also I'm wondering if message is the correct parameter name, it might be better as file_contents?